run/grpc-hello-world-mvn/README.md

```diff
@@ -0,0 +1,28 @@
+gRPC Kotlin Cloud Run Example
+-----------------------------
+
+Run Locally:
+1. In one shell / terminal window, start the server:
+    ```
+    ./mvnw compile exec:java -Dexec.mainClass="io.grpc.examples.helloworld.HelloWorldServerKt"
```

Comment:
I received the following error:

```
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.8.1:compile (default-compile) on project grpc-hello-world-kotlin: Compilation failure
[ERROR] /Users/betterbrent/Code/kotlin-samples/run/grpc-hello-world-mvn/target/generated-sources/protobuf/grpc-java/io/grpc/examples/helloworld/GreeterGrpc.java:[23,18] cannot find symbol
[ERROR]   symbol:   class Generated
[ERROR]   location: package javax.annotation
```

Reply:
this differs for jdk >= 9 or

the workaround is usually using a profile to determine the jdk version and whether this dependency is added or or not

Reply:
@saturnism Can you point me to an example?

Reply:
Oh, it looks like maybe our stub should depend on:
`javax.annotation:javax.annotation-api`

And then it'd work in JDK 8 & 9.  Alternatively, for now we can add the dep in the example.

Reply:
Added the dep to the sample for now.

Reply:
It works!